content/post/cell-based_architecture/index.md

```
@@ -0,0 +1,90 @@
+
+---
+
+### Footnotes:
+
+[^1]: [Slack’s Migration to a Cellular Architecture](https://slack.engineering/slacks-migration-to-a-cellular-architecture/)
+[^2]: [AWS Well-Architected](https://docs.aws.amazon.com/wellarchitected/latest/reducing-scope-of-impact-with-cell-based-architecture/what-is-a-cell-based-architecture.html)
+[^3]: Temporal Cloud - [Youtube video](https://www.youtube.com/watch?v=KvxAz5HwBpc)
+[^5]: [Staying in the Zone: How DoorDash used a service mesh to manage  data transfer, reducing hops and cloud spend](https://careers.doordash.com/blog/staying-in-the-zone-how-doordash-used-a-service-mesh-to-manage-data-transfer-reducing-hops-and-cloud-spend/)
+[^6]: [How We’re Making Roblox’s Infrastructure More Efficient and Resilient](https://blog.roblox.com/2023/12/making-robloxs-infrastructure-efficient-resilient/)
+[^7]: [Paper: Cell-Based Architecture](https://github.com/wso2/reference-architecture/blob/master/reference-architecture-cell-based.md) Fall 2024 version
+[^8]: [Cell-Based Architecture: Comprehensive Guide](https://dzone.com/articles/grokking-cell-based-architecture)
```

Markdownlint

Unused link or image reference definition: "^8"
Link and image reference definitions should be needed

(MD053, link-image-reference-definitions)